maths/prime_number.rb

```diff
@@ -0,0 +1,31 @@
+# A ruby program to check a given number is prime or not
+# Mathematical explanation: A number which has only 2 factors i.e., 1 (one) and itself
+
+# Prime number check function
+def is_prime_number(number)
+  prime_flag = false
+  if number>0
```

Comment:
**[Nits and Optional]**

I think it is more common to surround operators with spaces.

https://rubystyle.guide/#spaces-operators

```suggestion
  if number > 0
```